priv/perf/apps/load_test/lib/scenario/account_transactions.ex

```diff
@@ -0,0 +1,271 @@
+    {:ok, sender} = Account.new()
+    {:ok, _} = Faucet.fund_child_chain_account(sender, amount, @eth)
+
+    {:ok, faucet} = Faucet.get_faucet()
+
+    session
+    |> assign(faucet: faucet, iteration: 1)
+    |> wait_for_balance_update(sender)
+    |> log_info("user created: " <> Encoding.to_hex(sender.addr))
+    |> repeat(:repeat_task, [sender], iterations)
+    |> log_info("end...")
+  end
+
+  def wait_for_balance_update(session, sender, retry \\ @default_retry_attempts) do
+    {:ok, session} = do_wait_for_balance_update(session, sender, retry)
+    session
+  end
+
+  def repeat_task(session, sender) do
+    session
+    |> log_info("running iteration: " <> Integer.to_string(session.assigned.iteration))
+    |> retry_on_error(:test_apis, [sender], retries: @default_retry_attempts, random_delay: seconds(30))
+    |> update_assign(iteration: &(&1 + 1))
+  end
+
+  def test_apis(session, sender) do
+    session
+    |> test_api_account_get_balance(sender)
+    |> test_api_account_get_utxos(sender)
+    |> test_api_account_get_transactions(sender)
+    |> test_api_account_create_and_submit_transactions(sender)
+  end
+
+  defp test_api_account_get_balance(session, sender) do
+    start = Timing.timestamp()
+    {:ok, _} = get_balance(sender)
+
+    add_metric(
+      session,
+      {:call, {LoadTest.Scenario.AccountTransactions, '/account.get_balance'}},
+      Timing.timestamp() - start
+    )
+  end
+
+  defp test_api_account_get_utxos(session, sender) do
+    start = Timing.timestamp()
+    {:ok, _} = get_utxos(sender)
+
+    add_metric(
+      session,
+      {:call, {LoadTest.Scenario.AccountTransactions, '/account.get_utxos'}},
+      Timing.timestamp() - start
+    )
+  end
+
+  defp test_api_account_get_transactions(session, sender) do
+    start = Timing.timestamp()
+    {:ok, _} = get_transactions(sender)
+
+    add_metric(
+      session,
+      {:call, {LoadTest.Scenario.AccountTransactions, '/account.get_transactions'}},
+      Timing.timestamp() - start
+    )
+  end
+
+  defp test_api_account_create_and_submit_transactions(session, sender) do
+    start = Timing.timestamp()
+    {:ok, [inputs, sign_hash, typed_data, _txbytes]} = create_transaction(session, sender)
+
+    session =
+      add_metric(
+        session,
+        {:call, {LoadTest.Scenario.AccountTransactions, '/transaction.create'}},
+        Timing.timestamp() - start
+      )
+
+    typed_data_signed = sign_tx(inputs, sign_hash, typed_data, sender)
+
+    start = Timing.timestamp()
+    {:ok, response} = Api.Transaction.submit_typed(Connection.client(), typed_data_signed)
+
+    session =
+      add_metric(
+        session,
+        {:call, {LoadTest.Scenario.AccountTransactions, '/transaction.submit_typed'}},
+        Timing.timestamp() - start
+      )
+
+    %{
+      "txhash" => tx_id
+    } = Jason.decode!(response.body)["data"]
+
+    wait_until_tx_sync_to_watcher(session, tx_id)
+  end
+
+  defp create_transaction(session, sender) do
```

Comment:
Similarly to how it's done with child chain HTTP calls we can move all the functions that touch watcher API to modules `LoadTest.WatcherInfo.*` or to a single `LoadTest.WatcherInfo` module

Reply:
Agreed, but I'd prefer to do it in another PR